server/graphql/v1/mutations/expenses.js

```diff
@@ -445,11 +446,12 @@ async function payExpenseWithTransferwise(host, payoutMethod, expense, fees) {
   const [connectedAccount] = await host.getConnectedAccounts({
     where: { service: 'transferwise', deletedAt: null },
   });
-
   if (!connectedAccount) {
     throw new Error('Host is not connected to Transferwise');
   }
 
+  await handleTransferWisePaymentLimit(host, expense, { throwException: true });
```

Comment:
How should we handle the limitation on the frontend side?

Reply:
We can disable the button and display a tooltip like we're doing with the other plan limitations.
The backend error message shouldn't be too invasive tho.